src/app/plugins/checkclasses.plugin.ts

```diff
@@ -15,6 +15,10 @@ export default class CheckClassesPlugin extends Plugin {
 
   private _MAX_CHANS_SHOWN: number = 10;
 
+  public override validate(message: IMessage, args: string[]) {
+    return !!Moderation.Helpers.validateUser(args[0]);
```

Comment:
Bumping this again.
Need to check args.length
If no args are given args[0] will crash

Reply:
Oh sorry, I thought the !! took care of that. Added this check in, thanks!